src/lib/dto/v5/abstract-sensor-session.dto.v5.ts

```diff
@@ -1,11 +1,11 @@
-import { AbstractUserOwnedResponseV5 } from './abstract-user-owned.dto.v5';
 import { AbstractSensorEventCollectionResponseSummaryV5 } from './abstract-sensor-event.dto.v5';
+import { AbstractUserOwnedResponseV5 } from './abstract-user-owned.dto.v5';
 
 export interface AbstractSensorSessionResponseV5 extends AbstractUserOwnedResponseV5 {
 	competitionLevelUuid: string;
 	userUuid: string;
 	extensionType: string;
-	extensionDetails: { [name: string]: object };
+	extensionDetails: { [name: string]: any };
```

Comment:
I would avoid using the type `any` in favor of `unknown`. The type `any` won't produce any type errors which makes it tougher to debug. There's some further explanation here https://dmitripavlutin.com/typescript-unknown-vs-any/

If you found this in code I wrote, then boo me!

Reply:
Good to know, I didn't know unknown was a type!  Looks like you have extension details as a string type in the v3 code.  I was trying to convert from the Java V5 DTOs.  Should extensionDetails be a string or an object on the client?